mission-control/docs/installation/_properties_identity_mapper.mdx

```diff
@@ -0,0 +1,54 @@
+### Identity Mapper
+
+The schema for the object to be returned by the identity mapper script.
+
+<Fields rows={
+  [{
+    field: "role",
+    description: "Mission control role",
+    scheme: "string",
+  },
+  {
+    field: "teams",
+    description: "Teams the OIDC identity will be put into",
+    scheme: "`[]string`",
+  }]
+} />
+
+
+#### Kratos Identity
+
+<Fields rows={
+  [
+    {
+      field: "created_at",
+      description: "Timestamp for when the identity was created",
+      scheme: "timestamp",
+    },
+    {
+      field: "id",
+      description: "Unique identifier for the identity",
+      scheme: "string",
+    },
+    {
+      field: "metadata_public",
+      description: "Public metadata for the identity",
+      scheme: "`map[string]any`",
+    },
+    {
+      field: "state",
+      description: "Current state of the identity",
+      scheme: "`active` | `inactive`",
+    },
+    {
+      field: "traits",
+      description: "Identity's traits. Depends on the kratos configuration. See [default](https://github.com/flanksource/mission-control-chart/blob/main/chart/files/kratos-identity-schema.json).",
```

Comment:
I don't think the default schema will be changed by end_users, so we should include it here